pkg/resource/graph/dependency_graph.go

```diff
@@ -20,7 +20,8 @@ type DependencyGraph struct {
 // order with respect to the snapshot dependency graph.
 //
 // The time complexity of DependingOn is linear with respect to the number of resources.
-func (dg *DependencyGraph) DependingOn(res *resource.State, ignore map[resource.URN]bool) []*resource.State {
+func (dg *DependencyGraph) DependingOn(res *resource.State,
+	ignore map[resource.URN]bool, includeChildren bool) []*resource.State {
```

Comment:
I'm wondering whether this should be `includeChildren=true` always.  It's great to be conservative with the change here so we minimize blast radius, but we should follow up to figure this out what the implications here are..

Reply:
That was my initial instinct too, but was trying to reduce risk with the change. For what it’s worth, I tried running the unit tests with this always set to true and, somewhat terrifyingly, they passed. @pgavlin @justinvp What is the intended behavior here?